pandas/tests/series/test_arrow_interface.py

```diff
@@ -0,0 +1,23 @@
+import ctypes
+
+import pytest
+
+import pandas as pd
+
+pa = pytest.importorskip("pyarrow", minversion="16.0")
+
+
+def test_series_arrow_interface():
+    s = pd.Series([1, 4, 2])
+
+    capsule = s.__arrow_c_stream__()
+    assert (
+        ctypes.pythonapi.PyCapsule_IsValid(
+            ctypes.py_object(capsule), b"arrow_array_stream"
+        )
+        == 1
+    )
+
+    ca = pa.chunked_array(s)
+    expected = pa.chunked_array([[1, 4, 2]])
+    assert ca.equals(expected)
```

Comment:
Best to add a test case here specifying the type (to cover the `requested_schema` part). Something like:

```
arr = pa.array(s, type=pa.int32())
expected = pa.array([1, 4, 2], pa.int32())
assert arr.equals(expected)
```

Reply:
(but then using `chunked_array()` instead of `array()`, because `array(..)` actually doesn't work if we only define `__arrow_c_stream__` (and not `__arrow_c_array__`) ..